geomstats/geometry/pre_shape.py

```diff
@@ -513,6 +847,38 @@ def curvature(
         curvature = gs.reshape(curvature, max_shape)
         return curvature
 
+    def curvature_derivative(
```

Comment:
Here I am reluctant to use `tangent_vec_v` for the first argument because V would be a tangent vector in Kendall Shape space (which is by the way not defined) while it is by convention a vertical vector in the PreShapeSpace!

This need more thoughts, but I suspect that there will inevitably be collisions of names either withing classes / files or across heritage.

Reply:
Signatures for curvature derivatives and directional curvature derivatives were edited to use the uninformative letters a, b, c, d for tangent vectors. This requires an important work in the docstrings to explain to the potential user what is the mathematical formula that is really implemented so that the methods can be used without disassembling the code. Clashes in notations are now in the docstrings...